modules/ztp-image-based-upgrade-prep.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-prep_{context}"]
+= Preparing the {sno} clusters for the image-based upgrade
+
+At this stage, the {lcao} ensures that the nodes are ready for the upgrade by checking if the cluster meets certain conditions.
+You can complete this stage well before initiating the upgrade.
+
+After you initiate the preparation stage by using {cgu-operator}, {cgu-operator} applies a policy that starts the following procedures:
+
+* Installing and configuring the OADP Operator
+* Adding a `DataProtectionApplication` CR that points to the S3 backend
+* Adding a secret for the S3 credentials that are referenced in the `DataProtectionApplication` CR
+
+These resources are deleted after the upgrade is complete or if a rollback is initiated.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have installed {cgu-operator}.
+
+// are there other prereqs?
+
+.Procedure
+
+. Move to the next stage by changing the `Idle` value to `Prep` in the `ImageBasedUpgrade` CR.
++
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1alpha1
+kind: ImageBasedUpgrade
+metadata:
+  name: example-upgrade
+spec:
+  stage: Prep
+  seedImageRef:
+    version: 4.14.0
+    image: quay.io/<seed_image_repository>
+----
+// For telco, we need a sample with extraManifests and oadpContent
+
+. Apply the `ImageBasedUpgrade` policy with {cgu-operator}.
++
+[source,terminal]
+----
+$ oc apply -f example-upgrade.yml
+----
+// Please check command
+
+
+The {lcao} checks for the health of the cluster and pulls the seed image to the target cluster.
```

Comment:
"pulls the seed image to the target cluster" any details on this? is it cached somewhere?